samples/depthtesting.cpp

```diff
@@ -27,8 +27,8 @@
 
 #include <utils/EntityManager.h>
 
-#include "../samples/app/Config.h"
-#include "../samples/app/FilamentApp.h"
+#include "filamentapp/Config.h"
```

Comment:
Use `#include <filamentapp/Config.h>` here since we're including from a a public header? Same with all the other includes in the `samples/` directory.